src/content/en/updates/2017/09/chrome-62-media-updates.md

```diff
@@ -0,0 +1,173 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the media (audio/video) updates in Chrome 62.
+
+{# wf_updated_on: 2017-09-08 #}
+{# wf_published_on: 2017-09-08 #}
+{# wf_tags: news,chrome62,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Blink>Media #}
+
+# Media (Audio/Video) Updates in Chrome 62 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- [Offline playback with DRM licenses](#offline-playback) is now supported on
+  Android.
+- Chrome now [disables video tracks when a MSE video is played in the
+  background](#background-video-track-optimizations) to optimize performance.
+- Web developers can [customize seekable range](#seekable)
+  on live MSE streams.
+- Chrome now supports [FLAC in MP4 with MSE](#flac-in-mp4-with-mse).
+- Video will [go fullscreen when device is rotated](#auto-fullscreen-rotate).
+
+- TODO: Media preload defaults to metadata
```

Comment:
Does this belong here? What is it?

Reply:
The `TODO` means I'm still not sure it will be added in M62. If so, I'll remove the `TODO` and write a section for this.